docs/user-guide/content-map-toolbar.md

```diff
@@ -9,8 +9,10 @@ In particular, through this toolbar, the story editor is able to perform the fol
 
 * **Change media source** <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> accessing the [Media Editor](media-editor-window.md)
 
-* **Edit map configuration** through which is possible [Configure the map](configure-map.md)
+* **Edit map configuration** <img src="../img/button/map-configuration.jpg" class="ms-docbutton"/> through which is possible [Configure the map](configure-map.md)
 
 * **Change size** <img src="../img/button/change-size2.jpg" class="ms-docbutton"/> choosing between *Small*, *Medium*, *Large*
 
+* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the image. (It is available if the description is added to the image on the [Media Editor](media-editor-window.md))
```

Comment:
* **Hide caption** button <img src="../img/button/hide-caption.jpg" class="ms-docbutton"/> to show/hide the description under the **map**: this button is present only if a description has been provided for the map resource (see the [Media Editor](media-editor-window.md) tool for example).